genie-agent/build.gradle

```diff
@@ -0,0 +1,32 @@
+dependencies {
+    /*******************************
+     * Compile Dependencies
+     *******************************/
+
+    compile project(":genie-common")
+    compile project(":genie-proto")
+
+    // Logging
+    compile("org.slf4j:slf4j-api")
+
+    // Spring
+    compile("org.springframework.boot:spring-boot-autoconfigure")
```

Comment:
Redundant with the core starter unless you're trying to be explicit

Reply:
Done (copied from an example)